fe2-android/app/src/main/java/com/github/dedis/popstellar/ui/home/HomeActivity.java

```diff
@@ -279,7 +129,9 @@ public boolean onCreateOptionsMenu(Menu menu) {
   @Override
   public boolean onOptionsItemSelected(MenuItem item) {
     if (item.getItemId() == R.id.settings) {
-      mViewModel.openSettings();
+      Intent intent = new Intent(this, SettingsActivity.class);
+      Log.d(HomeViewModel.TAG, "Trying to open settings");
+      startActivity(intent);
```

Comment:
Most of the activity starting is done in ViewModel, is there a reason to do it here ?

Reply:
I missed this one